akka-cluster-tools/src/test/scala/akka/cluster/singleton/ClusterSingletonRestart2Spec.scala

```diff
@@ -42,6 +42,9 @@ class ClusterSingletonRestart2Spec extends AkkaSpec("""
       port = 0
     }
   }
+  akka.actor.serialization-bindings {
+    "akka.cluster.UniqueAddress" = jackson-cbor
```

Comment:
why override this? Comment?

Reply:
there is no serializer for UniqueAddress, not intended to be sent as a standalone message

comment added

Reply:
ah ofc